lib/ui/platform_dispatcher.dart

```diff
@@ -1314,6 +1371,100 @@ class WindowPadding {
   }
 }
 
+/// Area of the display that may be obstructed by a hardware feature.This is
+/// populated only on Android.
+///
+/// The [bounds] are measured in logical pixels. On devices with two screens the
+/// coordinate system starts with [0,0] in the top-left corner of the left screen
+/// and expands to include the visual space between the screens and the right screen.
+/// The [type] can be used to determine if this display feature obstructs the screen or not.
+/// For example, [DisplayFeatureType.hinge] and [DisplayFeatureType.cutout] both obstruct the display,
+/// while [DisplayFeatureType.fold] does not.
+class DisplayFeature {
+  const DisplayFeature({
+    required this.bounds,
+    required this.type,
+    required this.state,
+  });
+
+  /// Area of the view occupied by this display feature, measured in logical pixels.
+  ///
+  /// On devices with two screens, the Flutter view spans from the top-right corner
+  /// of the left screen to the bottom-right corner of the right screen, including
+  /// the area occupied by any display feature. Bounds of display features are
+  /// reported in this coordinate system.
+  ///
+  /// For example, on a dual screen device in portrait mode:
+  ///
+  /// - [bounds.left] gives you the size of left screen, in logical pixels.
+  /// - [bounds.right] gives you the size of the left screen + the hinge width.
+  final Rect bounds;
+
+  /// Type of display feature, e.g. hinge, fold, cutout.
+  final DisplayFeatureType type;
+
+  /// Posture of display feature, which is populated only for folds and hinges.
```

Comment:
This concept, especially with the randomly dropped terminology "posture" should have more explanation to avoid confusion if this is the entrypoint for developers reading the docs on this.